integrations/pom.xml

```diff
@@ -15,6 +15,7 @@
     <jenkins.version>2.164.1</jenkins.version> <!-- Remove when parent base is updated. -->
     <maven.deploy.skip>true</maven.deploy.skip> <!-- no need to be deployed during release, this is a test-only module -->
     <enforcer.skip>true</enforcer.skip> <!-- no need to be enforced, this is a test-only module -->
+      <rootdir>${basedir}/..</rootdir>
```

Comment:
indenting off here

isn't there a property build into maven you can use for this anyway?

Something like?
```
maven.multiModuleProjectDirectory
```

https://stackoverflow.com/questions/3084629/finding-the-root-directory-of-a-multi-module-maven-reactor-project

Reply:
rats, didn't even see indent was off. and no maven.multiModuleProjectDirectory is not there 😢 or at least intellij says it is not.

Reply:
Never trust your IDE 😞